src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,42 @@
 import React from 'react';
+import { User } from '../../types/User';
+import { Comment } from '../../types/Comment';
+import { CommentList } from '../CommentList';
+import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+type Props = {
+  title: string,
+  user: User,
+  body: string,
+  comments: Comment[] | null,
```

Comment:
It's better to use an empty array as a default value for comments prop instead of null